xCertificateServices/DSCResources/MSFT_xAdcsOnlineResponder/MSFT_xAdcsOnlineResponder.schema.mof

```diff
@@ -0,0 +1,8 @@
+[ClassVersion("0.1.0.0"), FriendlyName("xAdcsOnlineResponder")]
+class MSFT_xAdcsOnlineResponder : OMI_BaseResource
+{
+    [Required, Description("If the Online Responder service is configured to use Standalone certification authority, then an account that is a member of the local Administrators on the CA is required. If the Online Responder service is configured to use an Enterprise CA, then an account that is a member of Domain Admins is required."), EmbeddedInstance("MSFT_Credential")] String Credential;
+    [Write, Description("Specifies whether the Online Responder feature should be installed or uninstalled."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Key] String Name;
```

Comment:
No, this isn't possible. But in this case I followed the design pattern used in the existing xAdcsWebEnrollment resource which defines a name field that is used as the Key even though the field is not used at all.

I would have preferred to not have used this field at all, but because of the requirement that every DSC Resource contain at least one key, I stuck with what was done in the xAdcsWebEnrollment. Is there a more preferred method of handling this situation?